Swiftfin tvOS/Views/ServerDetailView.swift

```diff
@@ -35,36 +35,49 @@ struct EditServerView: View {
                     .frame(maxWidth: 400)
             }
             .contentView {
-                Section(L10n.server) {
-                    TextPairView(
-                        leading: L10n.name,
-                        trailing: viewModel.server.name
-                    )
+
+                Section {
+                    Button {} label: {
```

Comment:
We don't need just the server name to be focusable, but the user on the settings view did in order to have better scrolling to/from the tab bar.

Reply:
Leaving this as button still allows the cursor to go over it but it doesn't focus correctly. I changed this from a button to just the view and I believe this works as intended now.